### PR TITLE
[entt] Add feature natvis

### LINKED ISSUE #22162: [entt] Include natvis files
The library provides natvis files and an option to add them to the entt target, it would be nice to include them with the installation.

### PR DESCRIPTION
**Pull request by LilyWangLL**
Fixes #22162 
Add feature `natvis`, export `natvis` files.

### CODE REVIEW

File: ports/entt/fix-featureNatvis.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+index f47f9d3..740e16d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -277,6 +277,9 @@ install(
+ )
+ 
+ install(DIRECTORY src/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
++if(ENTT_HAS_NATVIS)
++	install(DIRECTORY natvis/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/natvis)
++endif()
```
Comment (JackBoosY) [inline ports/entt/fix-featureNatvis.patch:11]: @BillyONeal should we install the natvis files to the _include_ folder?

**Comment by BillyONeal** [comment]
This needs discussion/agreement among the vcpkg dev team that we want to use the indicated location in the installed tree. I don't think it'll be controversial but I don't expect to be able to get everyone to look at it until folks are back in the office (in the new year).

Assuming this is OK there will need to be edits to the maintainer guide to indicate the correct place.

**Comment by JackBoosY** [comment]
@BillyONeal Any result?

**Comment by BillyONeal** [comment]
I poked folks again today. We asked various debugger teams and their response was something to the effect of "why aren't your natvis files attached to the PDBs", and that still does make sense as the best possible experience.

Where that is impossible, for build system reasons or otherwise, I think we just need to pick a thing. I'm suggesting `share/${PORT}` as the right place. Also I don't think it should be a feature since it's just copying this extra file, we should just always do it.

**Comment by LilyWangLL** [comment]
Due to this commit is on my old reponsity, and this reponsity has been deleted. So I create a new PR #24675 fix this issue.

**Pull request closed without merging**